ksqldb-engine/src/main/java/io/confluent/ksql/query/QueryBuilder.java

```diff
@@ -191,10 +191,12 @@ TransientQueryMetadata buildTransientQuery(
         streamsBuilder
     );
 
-    final Map<String, Object> streamsProperties = buildStreamsProperties(
+    final Map<String, Object> streamsProperties = QueryBuilder.buildStreamsProperties(
```

Comment:
super nit: You don't need the `QueryBuilder.` thing. but at least have it consistent with line 280